src/lists/list-item.jsx

```diff
@@ -198,10 +198,6 @@ const ListItem = React.createClass({
         left: 16,
       },
 
-      primaryText: {
```

Comment:
this was my original PR, then I just though, the less dead code there is, the best it's.